### PR TITLE
Renaming  fake test buckets in HNS perf test

### PR DESCRIPTION
**Pull request by anushka567**
### Description
To avoid confusion created regarding unclaimed buckets mentioned in the test script ,  renamed all bucket names mentioned in the script to unit-test-mock-bkt to indicate that these buckets do not exist irl.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Manually ran the test script to confirm no failures.
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py [hunk 1]
```diff
@@ -240,14 +240,14 @@ def test_dir_already_exists_in_gcs_bucket(self, mock_list_directory):
   @patch("generate_folders_and_files._list_directory")
   def test_dir_does_not_exist_in_gcs_bucket(self, mock_list_directory):
     mock_list_directory.side_effect = [
-        ["gs://test_bucket/test_folder/", "gs://test_bucket/nested/"],
-        ["gs://test_bucket/test_folder/file_1.txt",
-         "gs://test_bucket/test_folder/file_1.txt"],
-        ["gs://test_bucket/nested/test_folder/"],
-        ["gs://test_bucket/nested/test_folder/file_1.txt"]
+        ["gs://unit-test-mock-bkt/test_folder/", "gs://test-bucket/nested/"],
```
Comment (ashmeenkaur) [inline perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py:243]:
```suggestion
        ["gs://unit-test-mock-bkt/test_folder/", "gs://unit-test-mock-bkt/nested/"],
```

File: perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py [hunk 2]
```diff
@@ -200,13 +200,13 @@ class TestCheckIfDirStructureExists(unittest.TestCase):
   @patch("generate_folders_and_files._list_directory")
   def test_dir_already_exists_in_gcs_bucket(self, mock_list_directory):
     mock_list_directory.side_effect = [
-        ["gs://test_bucket/test_folder/", "gs://test_bucket/nested/"],
-        ["gs://test_bucket/test_folder/file_1.txt"],
-        ["gs://test_bucket/nested/test_folder/"],
-        ["gs://test_bucket/nested/test_folder/file_1.txt"]
+        ["gs://unit-test-mock-bkt/test_folder/", "gs://test-bucket/nested/"],
```
Comment (ashmeenkaur) [inline perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py:203]:
```suggestion
        ["gs://unit-test-mock-bkt/test_folder/", "gs://unit-test-mock-bkt/nested/"],
```

File: perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py (hunk 1 above)
Comment (anushka567) [reply perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py]: Updated .

File: perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py (hunk 2 above)
Comment (anushka567) [reply perfmetrics/scripts/hns_rename_folders_metrics/generate_folders_and_files_test.py]: Updated.

**Review by ashmeenkaur — APPROVED** [review]

**Pull request merged**